src/auro-alaska.js

```diff
@@ -0,0 +1,162 @@
+  static get properties() {
+    return {
+      ...super.properties,
+      alaska: {
+        type: Boolean,
+        reflect: true
+      },
+      alaskaOfficial: {
+        type: Boolean,
+        reflect: true
+      },
+    };
+  }
+
+  /**
+   * @private async function to decide which logo to use
+   * @param {number} iconWidth size of parent icon is in
+   * @returns {object} SVG to be appended to DOM
+   */
+  alaskaLogoDef(iconWidth) {
+    switch (this.alaska) {
+
+      case iconWidth <= this.sm:
+        this.svg = new DOMParser().parseFromString(as100.svg, 'text/html').body.firstChild;
+        break;
+      case iconWidth >= this.sm + this.no && iconWidth <= this.md:
+        this.svg = new DOMParser().parseFromString(as200.svg, 'text/html').body.firstChild;
+        break;
+      case iconWidth >= this.md + this.no && iconWidth <= this.lg:
+        this.svg = new DOMParser().parseFromString(as300.svg, 'text/html').body.firstChild;
+        break;
+      default:
+        this.svg = new DOMParser().parseFromString(as400.svg, 'text/html').body.firstChild;
+    }
+  }
+
+  /**
+   * @private async function to decide which logo to use
+   * @param {number} iconWidth size of parent icon is in
+   * @returns {object} SVG to be appended to DOM
+   */
+  alaskaLogoTagDef(iconWidth) {
+    switch (this.alaskaOfficial) {
+
+      case iconWidth <= this.sm:
+        this.svg = new DOMParser().parseFromString(asTag100.svg, 'text/html').body.firstChild;
+        break;
+      case iconWidth >= this.sm + this.no && iconWidth <= this.md:
+        this.svg = new DOMParser().parseFromString(asTag200.svg, 'text/html').body.firstChild;
+        break;
+      case iconWidth >= this.md + this.no && iconWidth <= this.lg:
+        this.svg = new DOMParser().parseFromString(asTag300.svg, 'text/html').body.firstChild;
+        break;
+      default:
+        this.svg = new DOMParser().parseFromString(asTag400.svg, 'text/html').body.firstChild;
+    }
+  }
+
+  // lifecycle function for the purpose of
+  // displaying the correct Alaska logo
+  // with the correct Restricted icon
+  async firstUpdated() {
+    const iconContainer = await this.shadowRoot.querySelectorAll('.icon');
+    const iconWidth = iconContainer[this.zero].clientWidth;
+
+    if (this.alaskaOfficial) {
+      this.alaska = false;
+    } else {
+      this.alaska = true;
+    }
+
+    if (this.alaska) {
+      this.alaskaLogoDef(iconWidth);
+    } else if (this.alaskaOfficial) {
+      this.alaskaLogoTagDef(iconWidth);
+    }
+  }
+
+  static get styles() {
+    return [
+      super.styles,
+      css`${styleCss}`
+    ];
+  }
+
+  // function that renders the HTML and CSS into  the scope of the component
+  render() {
+    const classes = {
+      'icon': true,
+      'logo': this.alaska || this.alaskaOfficial,
+    }
+
+    return html`
+      <div class="${classMap(classes)}">
+        <div class="util_displayHiddenVisually">
+          <slot></slot>
```

Comment:
Do we need slots for `auro-alaska`? We should probably just define the visually hidden text in the component itself.

Reply:
I don't have the visually hidden text and this gives all the flexibility to the user of the CE. I get a lot of feedback that Auro should not be pushing content opinions. 

So, slot.